src/pages/register/index.jsx

```diff
@@ -51,31 +51,13 @@ function Register() {
       setDayValue('');
     }
   }, [monthValue, yearValue]);
-
   useEffect(() => {
-    if (
-      emailValue &&
-      userNameValue &&
-      passwordValue &&
-      monthValue &&
-      dayValue &&
-      yearValue
-    ) {
-      setDisabledButton(false);
-    } else setDisabledButton(true);
     if (monthValue && dayValue && yearValue) {
       setBirthDate(
         new Date(`${monthValue}/${dayValue}/${yearValue} EDT`).toISOString()
       );
     }
-  }, [
-    emailValue,
-    userNameValue,
-    passwordValue,
-    monthValue,
-    dayValue,
-    yearValue,
-  ]);
+  }, [monthValue, dayValue, yearValue]);
```

Comment:
We can exclude this effect and calculate the birth date directly when making a request